mysql/etc/containerbuddy.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
you are right, this stuff isn't being used, compose is pulling Tim's image from Docker Hub for this. I'll get rid of this extra stuff.